zeebe/protocol/src/main/java/io/camunda/zeebe/protocol/record/intent/ConditionalSubscriptionIntent.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright © 2017 camunda services GmbH (info@camunda.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.camunda.zeebe.protocol.record.intent;
+
+public enum ConditionalSubscriptionIntent implements ProcessInstanceRelatedIntent {
+  CREATED((short) 0),
+  TRIGGER((short) 1),
+  TRIGGERED((short) 2),
+  CANCELED((short) 3);
```

Comment:
❓ Is it possible to use `DELETED` instead of `CANCELED`, in order to consistency with the existing defined behavior?

*  `SignalSubscriptionIntent`
```
public enum SignalSubscriptionIntent implements Intent {
  CREATED((short) 0),
  DELETED((short) 1),
  MIGRATED((short) 2);
}
```

* `MessageSubscriptionIntent`
```
public enum MessageSubscriptionIntent implements ProcessInstanceRelatedIntent {
  ....
  DELETE((short) 6),
  DELETED((short) 7),
  ....
}
```